src/base/system_parameter.c

```diff
@@ -671,6 +671,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 #define PRM_NAME_REPL_LOG_GENERATOR_LOGGING "replication_log_generator_logging"
 #define PRM_NAME_REPL_LOG_LOCAL_DEBUG "replication_log_local_debug"
 
+#define PRM_NAME_REPL_ACK_ON_STREAM_FLUSH "replication_ack_on_stream_flush"
```

Comment:
What about having "replication_semisync_ack_mode"  parameter with enumeration values (on receive, on flush) ?
It may help if we reach the conclusion that we need yet another event when to send the ack.